Mathlib/Topology/MetricSpace/HausdorffDistance/Basic.lean

```diff
@@ -26,15 +25,37 @@ This files introduces:
 * `EMetric.hausdorffEdist s t`, the Hausdorff edistance of two sets in an emetric space
 * Versions of these notions on metric spaces, called respectively `Metric.infDist`
   and `Metric.hausdorffDist`
-* `Metric.thickening δ s`, the open thickening by radius `δ` of a set `s` in a pseudo emetric space.
-* `Metric.cthickening δ s`, the closed thickening by radius `δ` of a set `s` in a pseudo emetric
-  space.
+
+## Main results
+* `infEdist_closure`: the edistance to a set and its closure coincide
+* `EMetric.mem_closure_iff_infEdist_zero`: a point `x` belongs to the closure of `s` iff
+  `infEdist x s = 0`
+* `IsCompact.exists_infEdist_eq_edist`: if `s` is compact and non-empty, there exists a point `y`
+  which attains this edistance
+* `IsOpen.exists_iUnion_isClosed`: every open set `U` can be written as the increasing union
+  of countably many closed subsets of `U`
+
+* `hausdorffEdist_comm`: the Hausdorff edistance is symmetric
+* `hausdorffEdist_triangle`: the Hausdorff edistance satisfies the triangle inequality
+* ``hausdorffEdist_zero_iff_closure_eq_closure`: two sets have Hausdorff edistance zero
+  iff their closures coincide.
+* in particular, closed sets in an emetric space are an emetric space
+  (this is shown in `EMetricSpace.closeds.emetricspace`)
```

Comment:
That's too much stuff mentioned. Things like `hausdorffEdist_comm` can be readily found from their name.

Reply:
Fair point; I have shortened the docstring a bit.